de/LC_MESSAGES/django.po

```diff
@@ -8083,53 +8116,75 @@ msgstr "Weiterleitungen"
 msgid "Change Avatar"
 msgstr "Avatar ändern"
 
-#: kitsune/users/jinja2/users/edit_profile.html:21
+#: kitsune/users/jinja2/users/edit_profile.html:33 kitsune/users/jinja2/users/profile.html:20
+msgid "You can change your email address on Firefox Accounts."
+msgstr ""
+
+#: kitsune/users/jinja2/users/edit_profile.html:34 kitsune/users/jinja2/users/profile.html:21
+#, fuzzy
+msgid "Need help?"
+msgstr "Benötigt Änderungen"
+
+#: kitsune/users/jinja2/users/edit_profile.html:38
+msgid "Manage Firefox Account"
```

Comment:
Should `Account` be plural or singular?

Reply:
you should change it from origin source: https://github.com/mozilla/kitsune, I just extract all of it to here (for translations), We should not change typo at here because it still exist at source. By follow path: `kitsune/users/jinja2/users/edit_profile.html` at https://github.com/mozilla/kitsune/blob/master/kitsune/users/jinja2/users/edit_profile.html
Change you pull a request for change it? I think have many typo error, you should check, and I will change at origin source (kitsune) for you. Btw, I will pull a request for update translations per month? 
And you change about typo, grammar, vacabularies, etc... will update per month. is this ok?

Reply:
you should change it from origin source: https://github.com/mozilla/kitsune, I just extract all of it to here (for translations), We should not change typo at here because it still exist at source. By follow path: `kitsune/users/jinja2/users/edit_profile.html` at: https://github.com/mozilla/kitsune/blob/master/kitsune/users/jinja2/users/edit_profile.html
Change you pull a request for change it? I think have many typo error, you should check, and I will change at origin source (kitsune) for you. Btw, I will pull a request for update translations per month. And your change about typo, grammar, vacabularies, etc... will update per month. is this ok?